eng/Versions.props

```diff
@@ -2,7 +2,8 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>16.11.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
+    <VersionPrefix>16.11.0</VersionPrefix>
+    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
This change is really annoying but it'll keep happening so I'll try to take it (unless it merges cleanly in which case we'll have to back it out every time).